disk/tests/test_e2e.py

```diff
@@ -0,0 +1,20 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import sys
+
+import pytest
+
+from . import common
+
+
+@pytest.mark.e2e
+def test_check(dd_agent_check):
+    aggregator = dd_agent_check()
+    for metric in common.EXPECTED_METRICS:
+        aggregator.assert_metric(metric['metric'], metric_type=aggregator.GAUGE, device=metric['device'])
+    if sys.platform == 'darwin':
```

Comment:
This doesn't work for me, I get tmpfs instead of osxfs. Isn't that a bit weird to check it here anyway, as the check is running inside docker, but not the test?